include/ccf/js/registry.h

```diff
@@ -76,8 +76,20 @@ namespace ccf::js
      * Call this to populate the KV with JS endpoint definitions, so they can
      * later be dispatched to.
      */
-    void install_custom_endpoints(
-      ccf::endpoints::EndpointContext& ctx, const ccf::js::Bundle& bundle);
+    ccf::ApiResult install_custom_endpoints_v1(
```

Comment:
Ideally all need doxygen plus pointers in Sphinx.

Reply:
Doxygen'd now, I think we'll Sphinx later in a single pass.